interfaces/builtin/basedeclaration.go

```diff
@@ -154,6 +154,8 @@ plugs:
   snapd-control:
     allow-installation: false
     deny-auto-connection: true
+  unity8:
+    allow-installation: false
```

Comment:
Ah, I see you did add it to `restricted` in TestPlugInstallation. Not sure why I didn't see that before. Sorry.